src/Shared/AzureManaged/Shared.AzureManaged.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
There should not be a shared project. Rather, we should important the code directly into the both the client and worker projects. The other directories under `/src/Shared` as examples.

The reason for this is that we don't want to have to publish and maintain a "Shared" nuget package.

Reply:
Correct - move these files to `src/Shared/AzureManaged`, then include them via:

``` xml
    <SharedSection Include="AzureManaged" />
```

Reply:
i like number 2, 3 packages seem a bit cumbersome, basically we do a rename and go back to original approach before/ @cgillum for advice

Reply:
as discussed with Chris offline, he suggested to keep two packages and use differently named option types to separate shared ones